source/components/Modal.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+const warningText = `You have made changes to your map.
+ Editing the resolution or reimporting will overwrite those changes.`
+
+export default function Modal(props) {
```

Comment:
might be nice to give this class a more specific name (like `OverwriteWarningModal` or something), in case we want to factor out a common `Modal` in the future